dataland-frontend/tests/e2e/utils/GeneralUtils.ts

```diff
@@ -118,3 +118,26 @@ function checkIfContentIsIdentical(
     throwErrorBecauseOfFieldValue(newPath);
   }
 }
+
+/**
+ * This function opens a given dataset with its Multi Layer Data Table and checks wether a specific field can be hidden using the input switch.
+ * @param datasetIdentifier Identifies a dataset on the "my dataset" Page. The function will open the corresponding dataset.
+ * @param toggledFieldName Name of a field which is toggled by the input switch
+ */
+export function checkToggleEmptyFieldsSwitch(datasetIdentifier: string, toggledFieldName: string): void {
+  cy.wait(30);
+  cy.get("tr").contains(datasetIdentifier).click();
+  cy.get("span").contains(toggledFieldName).should("not.exist");
+  cy.get('div[data-test="hideEmptyDataToggle"]').should("exist");
+  cy.get('div[data-test="dataPointToggleButton"]')
+    .should("have.class", "p-inputswitch p-component p-inputswitch-checked")
+    .click();
+  cy.get('div[data-test="dataPointToggleButton"]').should("have.class", "p-inputswitch p-component");
```

Comment:
This check does not imply that these are all  classes present. I would suggest to replace all `.should("have.class", "p-inputswitch p-component p-inputswitch-checked")` with `.should("have.class", "p-inputswitch-checked")` and all `.should("have.class", "p-inputswitch p-component");` with `.should("not.have.class", "p-inputswitch-checked");`.